regexes/device/mobiles.yml

```diff
@@ -7527,6 +7586,32 @@ Zen:
   device: 'smartphone'
   model: '$1'
 
+# Lemhoov
+Lemhoov:
+  regex: 'Lemhoov'
+  device: 'smartphone'
+  models:
+    - regex: 'Lemhoov[ _-]([^/;]+) Build'
+      model: '$1'
+
+#Libra
+Libra:
+  regex: 'Libra[ _-]'
```

Comment:
Is the `Libra 97` maybe a product of the brand `GoClever`. Wasn't able to find it in another context...